metrics/scripts/enable.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+export INFLUX_HOST=http://localhost:8086
+export INFLUX_DATABASE=local
+export INFLUX_USERNAME=admin
+export INFLUX_PASSWORD=admin
+
+export SOLANA_METRICS_CONFIG="host=$INFLUX_HOST,db=$INFLUX_DATABASE,u=$INFLUX_USERNAME,p=$INFLUX_PASSWORD"
```

Comment:
how about just setting this env var, then running `scripts/configure-metrics.sh` to set all the INFLUX vars...less places to change in the future

Reply:
Seemed weird to call back down to configure-metrics.sh.  Maybe should move `configure-metrics.sh` and `metrics-write-datapoint.sh` to /metrics/scripts

Reply:
Moving seems better longer term but that's gonna cause some painful churn.  I vote to call down for now with a TODO in this file to consider moving those two scripts later